fundamentals/script.js

```diff
@@ -0,0 +1,274 @@
+const colorInput = document.querySelector('.color-selector');
+const widthInput = document.querySelector('.width-slider');
+const heightInput = document.querySelector('.height-slider');
+const colorBlock = document.querySelector('.color-block');
+const blockControlsWrapper = document.querySelector('.block-controls-wrapper');
+
+const TEST_DATA = {
+  ARRAY_OF_INTS: [1, 5, 6, 7]
+}
+
+var student = {
+  NAME : "David Rayy",
+  SCLAS : "VI",
+  ROLLNO : 12
+};
+
+/**
+ * TASK 1
+ *
+ */
+function getBiggerNumber(int1, int2) {
+  if ((int1 >= 40 && int1 <= 60) && (int2 >= 40 && int2 <= 60)) {
+    return int1 > int2 ? int1 : int2;
+  }
+
+  return 0;
+}
+
+/**
+ * TASK 2
+ *
+ */
+function getArraySum(intArr) {
+  return intArr.reduce(function (accumulator, currentValue) {
+    return accumulator += currentValue;
+  }, 0);
+}
+
+/**
+ * TASK 3
+ *
+ */
+function colorMixer() {
+  myColor = ["Red", "Green", "White", "Black"];
```

Comment:
Both single and double quotes are present in this function, please keep it consistent. Avoid using double quotes for strings, unless it's for string concatenation. We also talked about naming our variables. myColor should be a single color, not an array of colors.

Reply:
That is the way it was given in the Tasks 1 document my bad I had to change it.